### DISCUSSION TITLE
2024-05-29(수)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#62] [데일리스크럼]
아래 양식 복사해서 댓글로 작성해주시면 됩니다.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by Seoin02** [comment]
### 완료한 일
- 구글 로그인 구현
- 카카오 로그인 구현

### 해야할 일
- 마이(로그인x) UI 작업
- 약관 내용 정리
- 회원가입 UI 작업 시작

### 어려운 일

**Comment by devwqc** [comment]
### 완료한 일
- 레이아웃 초안 완료

### 해야할 일
- PC 화면에서 모달이 레이아웃을 이탈하는 현상 해결
- 카테고리 버튼 컴포넌트 구현
- 스크롤 Top 버튼 컴포넌트 구현
- 배너 캐러셀 컴포넌트 구현

### 어려운 일
- 생각보다 저희가 하는 레이아웃 구조가 어려운 거 같아요...! PC 화면에서 fixed를 사용하면 좌우 스크롤 했을 때 레이아웃을 이탈하는 현상이 있어서 고민이 많이 필요하네요🥲

**Comment by q45402sk** [comment]
### 완료한 일
Card 컴포넌트

### 해야할 일
Card 컴포넌트 코드리뷰 반영 
찜 컴포넌트 

### 어려운 일

**Comment by seolsis** [comment]
### 완료한 일
- 장바구니 css 
### 해야할 일
- 수량에 따른 가격 변동
- 장바구니 머지되면 결제 UI 작업 시작
### 어려운 일

**Comment by minyoung0503** [comment]
### 완료한 일

### 해야할 일
* 리뷰박스 컴포넌트 코드리뷰 반영
* 상품 상세 페이지 -> 리뷰 UI 작업

### 어려운 일
